.github/workflows/binary-verification.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 Kry10 Limited
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Can be called from a proof run to trigger binary verification.
+# Issues a repository-dispatch event on the graph-refine repository,
+# but only if there are c-graph-lang artifacts available.
+
+on:
```

Comment:
We should give a `name: ..` line. Or does that interfere with being called from other workflows?

Reply:
I think I just forgot to name it. Will fix.